apps/calendar/src/components/panel.tsx

```diff
@@ -114,18 +115,26 @@ export const Panel = forwardRef<HTMLDivElement, PropsWithChildren<Props>>(functi
     maxWidth,
   });
 
+  const isResizable = (category: string) => {
+    if (isNil(resizable) || isBoolean(resizable)) {
+      return !!resizable;
+    }
+
+    return resizable.includes(category);
+  };
+
```

Comment:
```suggestion
  const isResizable = useMemo(() => {
    if (isNil(resizable) || isBoolean(resizable)) {
      return !!resizable;
    }

    return resizable.includes(name);
  }, [resizable, name]);
```
`useMemo`로 감싸서 변수화하는건 어떨까요?

Reply:
네 요건 `useMemo` 쓰는게 더 적절해보이네요.